skforecast/utils/utils.py

```diff
@@ -1513,25 +1513,46 @@ def exog_to_direct_numpy(
 
 def expand_index(
     index: Union[pd.Index, None], 
-    steps: int
-) -> pd.Index:
+    steps: Union[int, str, pd.Timestamp],
+    **kwargs
+) -> Tuple[pd.Index, int]:
     """
     Create a new index of length `steps` starting at the end of the index.
     
     Parameters
     ----------
     index : pandas Index, None
         Original index.
-    steps : int
-        Number of steps to expand.
+    steps : Union[int, str, pd.Timestamp]
```

Comment:
Adapt to follow the same format as other arguments, "steps : int, str, pandas Timestamp"